### DISCUSSION TITLE
Better mobile support

### DISCUSSION THREAD

**Discussion by Sticks902** [GitbookIO/community#191] [Feature Requests]
### What's your feature/idea?

The usability of GitBook pages is scuffed on mobile devices. Better scaling for tables in particular would be appreciated. Adjusting the divisions of the table doesn't transfer to the mobile pages.
Even in horizontal view the menu won't close, which makes viewing even worse.
![image](https://user-images.githubusercontent.com/132195425/235345312-ee2e6ab2-38bc-44e8-aa5c-e62ff83140ca.png)
![image](https://user-images.githubusercontent.com/132195425/235345320-95b261d5-8072-4d0d-9ebe-967c4a86c154.png)
![image](https://user-images.githubusercontent.com/132195425/235345323-0fb990cc-fbd5-413b-a39d-69aaa5fa4f41.png)

### What's your desired outcome?

Better scaling to fit mobile resolutions.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by dot-alert** [comment]
Agreed 💯% on this.
Mobile support is a no-brainer in Web development, so there is no reason why mobile users should be forced to undergo such a terrible UX on GitBook. This is a let down for teams using GitBook to publish their technical docs and/or user guides.

**Comment by adelando** [comment]
Also 100% agree,

Most of the time, I'm unable to access a desktop to update a page. Having the ability to quickly add/edit a page on a mobile or tablet would be awesome, or even a dedicated app with all the features of a full desktop webpage.

On mobile, you can only use in landscape, but in portrait you don't get access to any keyboard

![image](https://github.com/GitbookIO/community/assets/11265974/5c8947e8-6ad6-4043-9d76-ffa8c7099284)

**Comment by aikoii** [comment]
Thanks all for sharing your thoughts! I'm not seeing problems with tables on published GitBooks when viewed on mobile, but could you point me to URLs where you have those issues or email us at support@gitbook.com?

Also, we're aware of the issues with editing on mobile devices. This is unlikely to be supported in a near future.
